tests/unit/swagger/templateHelpers.spec.ts

```diff
@@ -4,6 +4,62 @@ import * as templateHelpers from './../../../src/routeGeneration/templateHelpers
 
 describe('templateHelpers', () => {
 
+    afterEach(() => {
```

Comment:
hm.. if we were to call `ValidateParam` in the unit tests instead of directly calling `validateModel`, wouldn't that save is this `afterEach` and also the exports above?

Reply:
`models` is accessed by `validateArray` as well, which is directly called from the tests. This is where such a dependency might occur.

I will just refactor this, removing the shared state, no problem.

Reply:
Ah, you're right. I'm not sure those "sub"-validators should be exported. But thanks for fixing the shared state.

Reply:
Yeah, I'm always unsure as well. But I personally think in this case it makes sense to make them available so they can be tested individually because validation is quite complex for each case..